README

```diff
@@ -66,7 +66,6 @@ Example:
  XLIO INFO   : Node: r-aa-zorro006
  XLIO INFO   : ---------------------------------------------------------------------------
  XLIO INFO   : Log Level                      DEBUG                      [XLIO_TRACELEVEL]
- XLIO DETAILS: Log Details                    0                          [XLIO_LOG_DETAILS]
  XLIO DETAILS: Log Colors                     Enabled                    [XLIO_LOG_COLORS]
```

Comment:
Are those params still relavent? you mentioned that we do not have colors with DOCA
Don't doca provide a way to configure the logger?

Reply:
nope - you're right - removed anything to do with colors.